integration/internal/swarm/service.go

```diff
@@ -159,6 +159,14 @@ func ServiceWithEndpoint(endpoint *swarmtypes.EndpointSpec) ServiceSpecOpt {
 	}
 }
 
+// ServiceWithSysctls sets the Sysctls option of the service's ContainerSpec.
+func ServiceWithSysctls(sysctls map[string]string) ServiceSpecOpt {
```

Comment:
Is this only used in the unit-test?

Reply:
Yes (well, it's technically an integration test), but a lot of these functions are only used in one test.